server/src/main/java/org/opensearch/index/mapper/ConstantKeywordFieldMapper.java

```diff
@@ -0,0 +1,245 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
```

Comment:
Do you need this Blurb?  Could you check?

Reply:
new file so removed the elasticsearch license details